contents/tutorindigo.rst

```diff
@@ -0,0 +1,245 @@
+
+    @import "fonts"; 
+
+- Lastly, Set font family in ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_variables.scss`` file as below::
+
+    $font-family-title: 'Crimson', sans-serif;;
+    $serif: Crimson;
+    $f-sans-serif: Crimson, 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+
+The above steps need to be followed for CMS. But CMS requires one more step in addition to above steps for changing font family.
+
+- Include the below line in ``tutor-indigo/tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss`` file after ``$static-path`` definition::
+
+    $baseline: 20px;
+
+    $static-path: '..' !default;
+
+    @import "fonts";   /* add this line */
+
+    ......
+
+After doing so, Run ``tutor config save`` &  ``tutor local stop``. Run ``tutor images build openedx`` to build the Open edX image with the updated images. Finally, start the platform using ``tutor local start -d``. You can successfully view the updated font family using browser inspection tool as in the below image.
+
+.. list-table:: Font Updations Example
+   :widths: 50 50
+   :header-rows: 1
+
+    * - .. image:: /academy/static/images/tutorindigo/lms_font_change.png
+            :width: 400
+            :alt: LMS Inter Font Style Example
+      - .. image:: /academy/static/images/tutorindigo/cms_font_change.png
+            :width: 400
+            :alt: CMS Inter Font Style Example
+    
+
+Sass Styles Adjustments
+-----------------------
+
+Customize styles in ``tutorindigo/templates/indigo/lms/static/sass`` for LMS and ``tutorindigo/templates/indigo/cms/static/sass`` for CMS. Note that ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss`` file is used for adding and overriding styles. For Example, To change the background-color of body, follow the below steps:
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
+- Add your styles in the ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss``
+
+.. code-block:: sass
+
+    body {
+        min-height: initial;
+        background-color: #fff;
+    }
+
+- Install the updated Tutor Indigo Plugin
+
+
+Updating HTML templates
+-----------------------
+
+Add HTML files in ``tutorindigo/templates/indigo/lms/templates``, ensuring folder structure matches `edx-platform/lms/templates <https://github.com/openedx/edx-platform/tree/master/lms/templates>`_ for proper overriding.
+
+For example, You want to add detail of Open edX in footer. You have to follow the below points:
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
+- Search the template in `edx-platform/lms/templates <https://github.com/openedx/edx-platform/tree/master/lms/templates>`_  which is used for footer rendering.
+- The footer template exists at ``edx-platform/lms/templates/footer.html`` in edx-platform.
```

Comment:
These last two instructions should be grouped into one, because they are basically the same thing.